fossunited/api/reviewer.py

```diff
@@ -0,0 +1,311 @@
+import frappe
+
+from fossunited.api.dashboard import get_profile_data
+
+
+def get_event_cfp_submissions(event: str) -> list:
+    """
+    Get all the submissions for the given event
+
+    Args:
+        event (str): The id of the event
+
+    Returns:
+        list: List of submissions for the given event
+    """
+    fields = [
+        "name",
+        "linked_cfp",
+        "route",
+        "is_published",
+        "title",
+        "status",
+        "event",
+        "event_name",
+        "is_first_talk",
+        "session_type",
+        "talk_title",
+        "talk_reference",
+        "talk_description",
+        "custom_answers",
+        "positive_reviews",
+        "negative_reviews",
+        "unsure_reviews",
+        "approvability",
+    ]
+
+    is_cfp_anonymous = frappe.db.get_value(
+        "FOSS Event CFP", {"event": event}, "anonymise_proposals"
+    )
+
+    if not is_cfp_anonymous:
+        fields += [
+            "full_name",
+            "picture_url",
+            "designation",
+            "organization",
+            "bio",
+        ]
+
+    submissions = frappe.db.get_list(
+        "FOSS Event CFP Submission",
+        filters={"event": event, "status": "Review Pending"},
+        fields=fields,
+        order_by="creation desc",
+    )
+
+    return submissions
+
+
+@frappe.whitelist()
+def get_cfp_submissions_by_reviewer_status(
+    event: str,
+    status_filter: list = ["Reviewed", "Not Reviewed"],
+    to_approve_filter: list = ["Yes", "No", "Maybe"],
+):
+    if not has_reviewer_role():
+        frappe.throw("Unauthorized Access")
+
+    submissions_list = []
+
+    submissions = get_event_cfp_submissions(event)
+
+    reviewer = frappe.db.get_value(
+        "FOSS User Profile", {"user": frappe.session.user}, "name"
```

Comment:
Would be good if the constant USER_PROFILE could've been used here. In the last PR, the constant was added.

Ref: https://github.com/fossunited/fossunited/blob/ae7209ac0a8e54d046c46c73ac139817077cbba9/fossunited/doctype_ids.py#L2